pallets/automation-time/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "pallet-automation-time"
 description = "Pallet for scheduling and running tasks in the future."
 version = "0.0.0"
-edition = "2021"
+edition = "2018"
```

Comment:
2018?

Reply:
We haven't effectively switched to rust 2021. Depending on your toolchain, it might force you to update the rust config to `edition2021`. We should move to 2021 in a future pr (like soon)